develop/tutorials/articles/110-portlets/07-using-javascript-in-your-portlets/03-using-npm-in-your-portlets/05-how-the-bundler-formats-js-modules.markdown

```diff
@@ -79,24 +79,29 @@ The updated module code configured for AMD format is shown below:
 +$$$
 
 **Note:** The module's name must be based on its package, version, and file path 
-(for example `isobject@2.1.0/index`), otherwise Liferay AMD Loader can't find 
-it.
+(for example `my-bundle-package$isobject@2.1.0/index`), otherwise Liferay AMD 
+Loader can't find it.
 
 $$$
 
-Note the module's dependencies: `['module', 'require', 'isarray']`.
+Note the module's dependencies: 
+`['module', 'require', 'my-bundle-package$isarray']`.
 
 `module` and `require` must be used to get a reference to the `module.exports`
 object and the local `require` function, as defined in the AMD specification.
 
 The subsequent dependencies state the modules on which this module depends. Note 
 that `isarray` in the example is not a package, but rather an alias of the 
-`isarray` package's main module (thus, it is equivalent to `isarray/index`).
-
-Also note that @product@ has enough information in the `package.json` files 
-to know that `isarray` refers to `isarray/index`, but also that it must be 
-resolved to version `1.0.0` of such package, i.e., that `isarray/index` in this 
-case refers to `isarray@1.0.0/index`.
+`my-bundle-package$isarray` package's main module (thus, it is equivalent to 
+`my-bundle-package$isarray/index`).
+<!--
+Resolved packages no longer needed, so this paragraph can be removed
```

Comment:
@izaera Can this paragraph be removed? Since the 2.x bundler now uses namespaced packages, modules no longer need resolved, right?